storage/google/cloud/storage/bucket.py

```diff
@@ -1303,6 +1299,13 @@ def make_public(self, recursive=False, future=False, client=None):
                       ``NoneType``
         :param client: Optional. The client to use.  If not passed, falls back
                        to the ``client`` stored on the current bucket.
+
+        :raises ValueError:
+            If ``recursive`` is True, and the bucket contains more than 256
+            blobs.  This is to prevent extremely long runtime of this
+            method.  For such buckets, iterate over the blobs returned by
+            :meth:`Bucket.list_blobs` and call :meth:`Blob.make_public`
```

Comment:
I took a guess at these :meth: links. Will it link correctly?

Reply:
c8579d5